github/users.go

```diff
@@ -171,16 +171,16 @@ func (s *UsersService) GetContextualInfo(ctx context.Context, user string, opt *
 		return nil, nil, err
 	}
 
-	//TODO : remove custom Accept header when this API fully launches.
+	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeHovercardPreview)
 
-	contextResp := new(UserContextualInfoResponse)
-	resp, err := s.client.Do(ctx, req, contextResp)
+	uci := new(Hovercard)
```

Comment:
`uci` is no longer applicable. Make it `hc` or `hovercard`.

Reply:
Oops Sorry! missed it.
I will make the suggested improvements.